docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+services:
+  postgis:    
+    build:
+        context: docker/postgis
+    networks:
+      - default
+    ports:
+      - "5432:5432"
```

Comment:
you don't need to map 5432 to external port as mapfile should connect using postgis hostname directly

Reply:
I added it to view the database with my client (dbeaver).  But it is not necessary of course. Let me know and I changed

Reply:
I propose to remove it here and to document it as debug tips, what is your opinion here?

Reply:
I agree :) , i will change it